docs/src/main/asciidoc/tests-with-coverage.adoc

```diff
@@ -81,7 +80,7 @@ public class GreetingResource {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     @Path("/greeting/{name}")
-    public String greeting(@PathParam("name") String name) {
+    public String greeting(String name) {
```

Comment:
Why the `@PathParam` is gone here?

Reply:
It's part of the migration to RESTEasy Reactive: it's optional with it if the name of the parameter is the same. I had to keep it because we had some issues specific to JaCoCo but updating and fixing the quickstarts made it possible to get rid of it.

Reply:
See the whole cleanup I made here to use the `quarkus-jacoco` extension instead of JaCoCo instrumentation: https://github.com/quarkusio/quarkus-quickstarts/pull/1087/files .